app/views/teacher_training_adviser/steps/_identity.html.erb

```diff
@@ -6,7 +6,7 @@
 </p>
 
 <p>
-  To sign up, you'll need to have or be studying for a bachelor's degree, grade 2:2 or higher (or an equivalent).
+  To sign up you'll need to have, or be studying for, a bachelor's degree with honours. This needs to be a grade 2:2 or higher (or an equivalent).
```

Comment:
@EllieNodder I think this makes more sense now but still isn't quite right re class

If they have a degree - must be a 2:2 or above
If they don't have a degree, they need a predicted 2:2 or above **but only as a final year** - we don't ask this question to first years/second years/other

Should we just stick to saying 2:2 or higher i**f you have a degree already** here? Or try and explain further?